flake8_pie/pie805_prefer_simple_iterator.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+import ast
+import string
+from collections.abc import Iterable
+from typing import NamedTuple
+
+from typing_extensions import TypeGuard
+
+from flake8_pie.base import Error
+
+KNOWN_FUNCTIONS = {"items"}
+
+
+def is_name_list(val: Iterable[ast.expr]) -> TypeGuard[Iterable[ast.Name]]:
+    return all(isinstance(x, ast.Name) for x in val)
+
+
+class UsedVarRes(NamedTuple):
+    method: str
+    var: str
+
+
+def get_used_var(vars: Iterable[ast.Name]) -> UsedVarRes | None:
+    for idx, var in enumerate(vars):
+        if not var.id.startswith("_"):
```

Comment:
I think we should do proper usage tracking to avoid false positives

Reply:
pyflakes implement this usage tracking: https://github.com/PyCQA/pyflakes/blob/da197ee94e791640d82940396c19d2d82ca8defb/pyflakes/checker.py#L1207